### PR TITLE
add account support

### PR DESCRIPTION
**Pull request by selsta**
Updated and rebased version of #793 

Known bugs:

- Copied addresses are truncated. (monero-project/monero#5074)
- Account table buttons are kinda buggy. (#1907)

I’d appreciate feedback!

### CODE REVIEW

**Comment by selsta** [comment]
Update:

acc

**Comment by selsta** [comment]
Ready for reviews now.

File: pages/Account.qml [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+                        }
+                        onExited: {
+                            parent.color = MoneroComponents.Style.white
+                        }
+                        onClicked: {
+                                console.log("Copied to clipboard");
+                                clipboard.setText(parent.text);
+                                appWindow.showStatusMessage(qsTr("Copied to clipboard"),3)
+                        }
+	                }
```
Comment (mmbyday) [inline pages/Account.qml]: please fix indentation

File: pages/Account.qml [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
```
Comment (mmbyday) [inline pages/Account.qml]: please fix indentation

File: pages/Account.qml (hunk 1 above)
Comment (selsta) [reply pages/Account.qml]: Copy pasting from Github diffs adds a tab :/ Thanks.

**Pull request merged**